### PR TITLE
Sit cfg sometimes needs domain name fix

### PR DESCRIPTION
**Pull request by tbarnes-us**
Fix for pulls #797 and #820:   the calculated DOMAIN_NAME value in introspectDomain.py should be discovered via wlst instead of from a domain env var.   @alai8 discovered that the domain env var isn't necessarily in sync with the 'actual' value.

### CODE REVIEW

File: site/weblogic-docker-images.md [hunk 1]
```diff
@@ -9,8 +9,8 @@ There are two main options available:
   and the domain directory.
 
 If you want to use the first option, you will need to obtain the standard
-WebLogic Server image from the Docker Store [see here](#obtaining-standard-images-from-the-docker-store)
-and then create a new image with the mandatory patches applied as described in [this section](#creating-a-custom-images-with-patches-applied).
+WebLogic Server image from the Docker Store, [see here](#obtaining-standard-images-from-the-docker-store),
+this image already contains the mandatory patches applied as described in [this section](#creating-a-custom-image-with-patches-applied).
```
Comment (tbarnes-us) [inline site/weblogic-docker-images.md]: Not sure about this change - the merge of the develop produced this conflict somehow...   I suspects 'master' might be more up to date, and the above matches master?

File: site/weblogic-docker-images.md [hunk 2]
```diff
@@ -64,12 +68,16 @@ Docker Store.
 
 ## Creating a custom image with patches applied
 
+<<<<<<< HEAD
```
Comment (alai8) [inline site/weblogic-docker-images.md]: git merge conflict? There is another one above.

Comment (tbarnes-us) [reply site/weblogic-docker-images.md]: I just pushed a merge.  Not quite sure if I did it right.  Not sure if 'master' or 'develop' is supposed to be the lead for the change.   Checking.

Comment (tbarnes-us) [reply site/weblogic-docker-images.md]: I just pushed the version that was on develop.

Comment (tbarnes-us) [reply site/weblogic-docker-images.md]: So they should match now.

**Pull request closed without merging**